peachjam/js/components/FindDocuments/index.vue

```diff
@@ -927,6 +931,18 @@ export default {
       }
     },
 
+    loadSaveDocumentButtons () {
+      // use htmx to load and inject save-document buttons
+      // get document ids
+      const ids = this.searchInfo.results.map(result => result.id);
+      if (ids.length) {
+        const el = document.createElement('div');
```

Comment:
htmx wants an element to inject content into, so we create this. in reality, all the content is swapped into place by htmx, but i add this to the end of the document just to be safe.